### ISSUE TITLE
Spreadsheet of all molecules evaluated to date

### ISSUE THREAD

**Issue by mattodd** [StructuralGenomicsConsortium/CNP14-WDR91#1]
HI @KangpingL @Mohamadshahir1 can you please post the shared, online sheet of all molecules in this project so far evaluated vs WDR91. We need an editable Excel sheet. This needs to be linked on the wiki under "sources of data". Also inlcude the link below as a comment on this Issue. Make sure it's set to "anyone with the link can view", with edit rights given to select people (including me...). You can close this Issue with a comment when done.

**Comment by Mohamadshahir15** [comment]
Hello, here is the link to the summary of the molecule. [Summary of molecule](https://liveuclac-my.sharepoint.com/:x:/r/personal/ucnv305_ucl_ac_uk/Documents/Summary%20of%20molecule.xlsx?d=w2044feeb95c64295b9c3ffa11bc15928&csf=1&web=1&e=5DpjoW)

**Comment by mattodd** [comment]
OK, thanks. Is this file in some way linked on the wiki, in the Story So Far page? And can you add the Excel file to a comment below, so we have another copy. @KangpingL is it up to date?

**Comment by KangpingL** [comment]
> OK, thanks. Is this file in some way linked on the wiki, in the Story So Far page? And can you add the Excel file to a comment below, so we have another copy. @KangpingL is it up to date?

Hi Mat, no, this googledoc does not include the 20 molecules we bought earlier.
@Mohamadshahir15 I have added the SMILES and KDs of those 20 bought molecules in the shared file to make your life easier. Can you add their corresponding InChI in the file please?

**Comment by Mohamadshahir15** [comment]
Yes, it is on the Story So Far page. Thank you @KangpingL. Will complete it ASAP and attach the file to the comment.
